bin/install-carthage

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env sh
+
+version=0.9.4
+
+curl -L -O "https://github.com/Carthage/Carthage/releases/download/$version/Carthage.pkg"
```

Comment:
I wonder if this is subject to rate limits.

I ran into something with Travis and Carthage binaries. I know that's using a different mechanism but it seemed to be checking the IP address.